project-root/Assets/Scripts/NavMeshSpawner.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.AI;
+
+public class NavMeshSpawner : MonoBehaviour
+{
+    public GameObject monsterSpawn;
+    Vector3 point;
+    public float time;
+    public bool done;

```

Comment:
This is an example of a bad variable name. What is done? Be more specific when it comes to naming your variables, you may know what they mean but someone else reading your code may not know.